<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CredentialDialog</class>
 <widget class="QDialog" name="CredentialDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lblMessage">
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblCredential"/>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cmbCredential"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblUsername"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="txtUsername"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblPassword"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="txtPassword">
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="btnUse"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
